README.md

```diff
@@ -19,74 +19,94 @@ These are related projects that are either already setup and available for use o
 * [Dockstore](https://dockstore.org): our workflow and tool sharing platform
 * [Toil](https://github.com/BD2KGenomics/toil): our workflow engine, these workflows are shared via Dockstore
 
-## Installing the Platform
 
-These directions below assume you are using AWS.  We will include additional cloud instructions as `cgp-deployment` matures.
```

Comment:
I would think we would eventually want to be able to run on other platforms. I think you should keep this.

Reply:
Ah, I now see you moved this further below. I would think you would want to keep it towards the top of the file, before the AWS specific instructions, and not after the AWS-specific instructions.

Reply:
Put it back into that place.